src/libraries/Directory.Build.props

```diff
@@ -85,6 +85,10 @@
     <EnableCoverageSupport Condition="'$(ContinuousIntegrationBuild)' != 'true'">true</EnableCoverageSupport>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TestReadyToRun)' == 'true'">
```

Comment:
There's zero references to TestReadyToRun in src/libraries. Should all of TestReadyToRun handling stay in eng/? tests.singlefile.targets looks more suitable.

Reply:
I tried moving it, but then the build fails in other subsets because it can't find the `apphost`. Like for example, this one:

```text
/path/to/runtime/.dotnet/sdk/8.0.100-rtm.23506.1/Microsoft.Common.CurrentVersion.targets(5189,5): error MSB3030: Could not copy the file "/path/to/runtime/artifacts/obj/System.Runtime.Tests/Release/net9.0-unix/apphost" because it was not found. [/path/to/runtime/src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj::TargetFramework=net9.0-unix]
    0 Warning(s)
    1 Error(s)
```

In this case, this means `UseLocalAppHostPack` is set to false, i.e., it is not using it when building the tests and that's where we need it. We don't need it for `clr` or `libs`, just `libs.tests`. So I'd like to ask if you had another possible solution, if adding it to `Directory.Build.props` is not right.